homework/CSharpOOP3/Pyramid.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP3
+{
+    internal class Pyramid : Shape
+    {
+        private string _name;
```

Comment:
I would recommend you to create public property Name where you can get the value of _name filed in the Shape class and use the property in all inheritance classes.